exploration/syntax-exploration-2.md

```diff
@@ -0,0 +1,257 @@
+# Syntax Exploration v2
+
+In the 2023-10-16 teleconference, we narrowed the options being considered for text-mode syntax to "1a", "2a", and "3a". 
+Each of these syntax candidates has been adjusted based on our conversation.
+
+One open issue, separately considered here, is the use of an additional sigil for some of the syntaxes.
+We have generally used `#` as the placeholder for this sigil in the previous iteration.
+
+
+## Comparison Matrix
+
+| Option | Description                                                    | Doesn’t Nest {} | Doesn’t Need More Escapes | Doesn’t Require Quoted Pattern |  Counted {} works | Multiple Expression Syntaxes |
+| :----- | :------------------------------------------------------------- | :-------------- | :------------------------ | :----------------------------- |  :--------------- | :------------ |
+| 1a     | Invert for text mode, distinguish statements from placeholders | -               | +                         | +                              | +                | - |
+| 2a     | Text first, current syntax for complex messages                | -               | +                         | -                              | -                | + |
+| 3a     | Use sigils for code mode                                       | +               | -                         | +                              | +                | + |
```

Comment:
```suggestion
| Option | Description                                                    | Doesn’t Nest {} | Doesn’t Need More Escapes | Doesn’t Require Quoted Pattern |  Counted {} works | Multiple Expression Syntaxes | Amt of Complexity Added | Legibility of single-line
| :----- | :------------------------------------------------------------- | :-------------- | :------------------------ | :----------------------------- |  :--------------- | :------------ | :-------------- |  :--------------
| 1a     | Invert for text mode, distinguish statements from placeholders | -               | +                         | +                              | +                | - |            | - |            | - |
| 2a     | Text first, current syntax for complex messages                | -               | +                         | -                              | -                | + |            | + |            | + |
| 3a     | Use sigils for code mode                                       | +               | -                         | +                              | +                | + |            | - |            | - |
```

Reply:
@echeran The table was meant to include mainly objective differences between the candidates. I think "complexity added" and "legibility of single-line" are somewhat subjective. Do you agree?

The one part of this table that should probably change in that light is "doesn't require quoted pattern" should probably be "allows unquoted pattern" and I will make that change in a second. There are strong arguments for why quoted-pattern _is a feature_ and not something to avoid, which I expect proponents of `2a` will include in their thinking for why they like that candidate.